test/unit/search_parameter_test.rb

```diff
@@ -0,0 +1,43 @@
+require "test_helper"
+require "search_parameters"
+
+
+class SearchParameterTest < ShouldaUnitTestCase
+
+	context "quoted_search_phrase?" do
+		should "return false if query isn't quote enclosed" do
+			params = SearchParameters.new(query: "my query")
+			assert_equal false, params.quoted_search_phrase?
+		end
+		
+		should "return false if entire query isn't quote enclosed" do
+			params = SearchParameters.new(query: %Q{This is "part of my" query} )
+			assert_equal false, params.quoted_search_phrase?
+		end
+
+		should "return false if query doesn't have ending quotes" do
+			params = SearchParameters.new(query: %Q{"unclosed quotes} )
+			assert_equal false, params.quoted_search_phrase?
+		end
+
+		should "return false if query doesn't have starting quotes" do
+			params = SearchParameters.new(query: %Q{unclosed quotes"} )
+			assert_equal false, params.quoted_search_phrase?
+		end
+
+		should "return true if query enclosed with quotes" do
+			params = SearchParameters.new(query: %Q{"phrase enclosed with quotes"} )
+			assert_equal true, params.quoted_search_phrase?
+		end
+
+		should "return true if query enclosed with quotes but with leading whitespace" do
+			params = SearchParameters.new(query: %Q{  \t  "phrase enclosed with quotes"} )
+			assert_equal true, params.quoted_search_phrase?
+		end
+	
+		should "return true if query enclosed with quotes but with trailing whitespace" do
+			params = SearchParameters.new(query: %Q{"phrase enclosed with quotes"  \t  } )
+			assert_equal true, params.quoted_search_phrase?
+		end
+	end
+end
```

Comment:
Please use soft tabs, here is our styleguide: https://github.com/alphagov/styleguides/blob/master/ruby.md#general